_includes/prepare_conference_data.liquid

```diff
@@ -0,0 +1,7 @@
+{% assign conferences = nil %}
```

Comment:
Thanks for giving this a short @komarovb!

Looking at this, I'm wondering if this doesn't make things more complicated. 
Maybe we could also think about the way we show things. 

Since we have the conferences split up per year now, maybe we should also have dedicated pages per year to look at past conferences. This would also simplify this part since we don't have to concatenate the conferences.

What do you think?

Reply:
Hey @marcoroth, thank you for looking at this.

I agree that the implementation is a bit trickier now, but the main idea, as far as I understand, is to make it easier for people to add their events to the website and avoid navigating through the 2.5k+ lines of the conferences file.

In terms of user experience, I personally like how the website feels at the moment. The home page is clear and concise, showing not only current conferences but also future ones. One simplification that comes to mind is to drop concatenating all the files and just use the current and next year's data for the home and CFP pages.

Regarding past conferences, it might be beneficial to show them grouped by year, perhaps using an accordion to simplify navigation to a particular year. Here is a primitive example:

<img width="648" alt="Screenshot 2024-07-04 at 23 13 20" src="https://github.com/ruby-conferences/ruby-conferences.github.io/assets/10147456/14208fe9-ae80-4e6c-bcfd-9dffc5f21aea">

I could prototype something to give it a feel, and we can see if it's useful.

Let me know what you think.